src/openapi/generators.ts

```diff
@@ -130,6 +133,15 @@ const generateRequestBodySchema = (
     'text',
     undefined,
   )
+
+  schema.required = ((schema.required as string[]) ?? []).filter(
+    property => !['id', 'createdAt', 'updatedAt'].includes(property),
```

Comment:
```suggestion
    property => schema.properties[property] !== undefined
```

I believe this makes the intent clearer and also avoids repetition

Reply:
Yeah, I think that makes sense, please do that :slightly_smiling_face:

Reply:
Done https://github.com/janbuchar/payload-oapi/pull/57/commits/85df912589605d7106b7340f712f4cca63cd5038